netbench/netbench-driver/Cargo.toml

```diff
@@ -28,4 +28,4 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 #
 # NOTE: The version of the `openssl-sys` crate is not the same as OpenSSL itself.
 #       Versions 1.0.1 - 3.0.0 are automatically discovered.
-openssl-sys = { version = "<= 0.9", features = ["vendored"] }
+openssl-sys = { version = "0.9", features = ["vendored"] }
```

Comment:
what's the reason for this change?

Reply:
It was already done in s2n-quic-tls after we started supporting OpenSSL 3.0
https://github.com/aws/s2n-quic/blob/ac601d799faffd2eb8b75caec101bf0befd90651/quic/s2n-quic-tls/Cargo.toml#L34

I just forgot to apply it to the netbench config.